src/Badge.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+const type = props => {
+  switch(props.type) {
+    case 'success':
+      return {
+        backgroundColor: theme.colors.green,
+        color: theme.colors.white
+      }
+    case 'success-light':
+      return {
+        backgroundColor: theme.colors.green0,
+        color: theme.colors.black
+      }
+    case 'warning':
+      return {
+        backgroundColor: theme.colors.red,
+        color: theme.colors.white
+      }
+    case 'warning-light':
+      return {
+        backgroundColor: theme.colors.red0,
+        color: theme.colors.black
+      }
+    case 'attention':
+      return {
+        backgroundColor: theme.colors.orange,
+        color: theme.colors.white
+      }
+    case 'attention-light':
+      return {
+        backgroundColor: theme.colors.orange0,
+        color: theme.colors.black
+      }
+    case 'information':
+      return {
+        backgroundColor: theme.colors.blue,
+        color: theme.colors.white
+      }
+    case 'information-light':
+      return {
+        backgroundColor: theme.colors.blue0,
+        color: theme.colors.black
+      }
+    default:
+      return {
+        backgroundColor: theme.colors.blue,
+        color: theme.colors.white
+      }
+  }
+}
+
+
+const Badge = styled.div`
+  border-radius: 99999px; 
+  display: inline-block;
+  font-size: ${props => props.theme.fontSizes[0]}px;
+  font-weight: 600;
+  ${type}
+  ${space}
+`
+
+Badge.defaultProps = {
+  px:2,
+  py:1
+}
+
+const numberStringOrArray = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.string,
+  PropTypes.array
+])
+
+Badge.propTypes = {
+  type: PropTypes.string,
```

Comment:
Would like to understand the your thoughts behind `type` prop a little bit more. But currently, the `type` prop is tied to keywords such as `success`, or `warning`, to determine background color and color of the label inside the badge.

Found these keywords somewhat confusing. If my use case for the `Badge` is not related to any severity level, but instead, let's say, my website is about food and my UI changes the badge's background color and color of the label by type of cuisine. In such use case, type of cuisine does not fit the current keywords such as `success` and etc.

Perhaps we can just simplify the value of types to use numeric, i.e.:

- `<Badge type={1} />` => light blue background with black label

- `<Badge type={2} />` => dark blue background with white label

and so forth.

It'd be great if this PR can also update the documentation space to provide a README for `Badge` that explain the `type` prop design with use cases.

###### Documentation Space
https://github.com/pricelinelabs/design-system/tree/master/docs

###### README page with links to documentation topics
https://github.com/pricelinelabs/design-system/blob/master/README.md

As your 1st `Badge` story, you can also consider a self-documentation self with your code using `withInfo(...)`.
###### Story Code
https://github.com/pricelinelabs/design-system/blob/master/.storybook/Heading.js#L11

###### Heading Documentation Generated By `withInfo(...)` story.
https://pricelinelabs.github.io/design-system/?selectedKind=Heading&selectedStory=Heading%20component&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel

Reply:
@Degron Looks like you refactored this `type` out and are using `bg`. Please remove the old prop from prop-types and add `bg` instead.